source/contracts/reporting/ReportingWindow.sol

```diff
@@ -234,7 +236,11 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
         IUniverse _shadyUniverse = _shadyReportingWindow.getUniverse();
         require(_shadyUniverse.isContainerForReportingWindow(_shadyReportingWindow));
         require(universe.isParentOf(_shadyUniverse));
-        _cash.transfer(_shadyReportingWindow, _balance);
+        IUniverse _destinationUniverse = _shadyUniverse;
+        IReportingWindow _destinationReportingWindow = _shadyReportingWindow;
+        bytes32 _winningForkPayoutDistributionHash = universe.getForkingMarket().getFinalPayoutDistributionHash();
+        require(_destinationUniverse == universe.getOrCreateChildUniverse(_winningForkPayoutDistributionHash));
```

Comment:
I think there is a `getChildUniverse` that is constant and throws if the universe doesn't exist.  Should be safe to call that one instead (less gas) since we will never be migrating fees to a universe that hasn't been created yet.